### ISSUE TITLE
Handle custom protocol (ytmd)

### ISSUE THREAD

**Issue by guaycuru** [ytmdesktop/ytmdesktop#670]
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [X] I verified that there is no open issue for the same subject.

I think it'd be very interesting to make the app handle a custom protocol. A high level look through the code showed me that the app already registers itself to handle the  `ytmd://` protocol, but I couldn't find any mention on how to use it. What is the full URL supposed to be like to make the app load (or even start playing) a specific album or song?

**Comment by Alipoodle** [comment]
I do wish to get this Documented into the Wiki BUT this is how it works as of the Dev Build (there's a fix for some stuff in there)

URI Protocol: `ytmd://`

# Basic Documentation
### Setttings
`ytmd://settings`
This will open Settings of YTM Desktop

### Play (note this is broken on v1.13 & v1.14)
`ytmd://play/{VideoId}`
This will open the Video into YTM Desktop

`ytmd://play/{VideoId}/{PlaylistId}`
This will open a Video as well as a Playlist (this could be an Album or something)

For Example the URL for **Koven**'s **Butterfly Effect (VIP)** from the **Butterfly Effect Deluxe** Album is https://music.youtube.com/watch?v=D9LS1MWs2os&list=OLAK5uy_kGlJZOdaZ_mysR7DUioGPvEq1am5gy5Xc
If I wanted to run a URI to open this I would do `ytmd://play/D9LS1MWs2os/OLAK5uy_kGlJZOdaZ_mysR7DUioGPvEq1am5gy5Xc`

--------------------
# Handle
This is handled all within the `handleOpenUrl()` in main.js of the application.
[Master Branch](https://github.com/ytmdesktop/ytmdesktop/blob/master/main.js#L1679) | [Development Branch](https://github.com/ytmdesktop/ytmdesktop/blob/development/main.js#L1765)

**Comment by guaycuru** [comment]
Oh that's interesting, thank you very much.
Although there seems to be a problem with loading a video by id. I just got this exception:

```
Uncaught Exception:
ReferenceError: loadMusicByVideoId is not defined
```

I'll try to open a PR fixing that.

**Comment by Alipoodle** [comment]
![2021-03-22_17-43-08](https://user-images.githubusercontent.com/17199186/112034311-3e05e280-8b36-11eb-8446-1c74ea82909a.gif)

**Comment by Alipoodle** [comment]
> Oh that's interesting, thank you very much.
> Although there seems to be a problem with loading a video by id. I just got this exception:
> 
> ```
> Uncaught Exception:
> ReferenceError: loadMusicByVideoId is not defined
> ```
> 
> I'll try to open a PR fixing that.

As I said within my thing, this is already fixed within the Development build and why I put that it doesn't work in 1.13 and 1.14.
This is fixed in e54343c662b188196f11791e03b8c727d43c3218 / bb6ff6b630477238c7dc3e2fb592ac81ef7cbc88

**Comment by guaycuru** [comment]
Oh ok sorry, I already edited my comment.
Thank you very much :)
